frontend/src/components/auth/login/login.css

```diff
@@ -66,6 +65,14 @@
   box-shadow: inset 2px 2px 5px #888888, inset -2px -2px 5px #ffffff;
 }
 
+.inputLogin:focus{
+  box-shadow: inset 3px 2px 5px #050142, inset -3px -2px 5px #e005d5;
```

Comment:
change this to `box-shadow: inset 2px 2px 5px #1E488F, inset -2px -2px 5px #1E488F;`